Makefile

```diff
@@ -41,6 +41,17 @@ IMAGE_TAG_BASE ?= $(IMAGE_REPO)/operator
 # TAG allows the tag for the operator image to be changed. Defaults to the VERSION
 TAG ?= $(VERSION)
 
+# IMAGE_PUSH indicates if any of the images should be pushed. By default, images are not pushed
+# unless a command indicates that a push occurs.
+IMAGE_PUSH ?= false
+
+# Format for the SBOM produced by ko.
+# Defaults to "spdx", use "none" to disable SBOM generation
+SBOM ?= "spdx"
+
```

Comment:
ko may change its default ~soon, to either cyclonedx, or (more likely) both. Just something to keep in mind here.